themes/classic/templates/checkout/_partials/steps/shipping.tpl

```diff
@@ -79,6 +79,15 @@
             </div>
           {/block}
           <div class="order-options">
+            {if $separatePackages.allowed}
+              <span class="custom-checkbox">
+              <input class="js-gift-checkbox" id="allow_separated_package" name="allow_separated_package"
```

Comment:
@thalassa-web can we change this class? I think it should be related to the separated package checkbox

Reply:
After some discussion with myself and I … You're right !